docs/ini-file-deduction.md

```diff
@@ -0,0 +1,32 @@
+# Deduction of the Core INI Format
+
+Since the INI-file format isn't specified yet, the features of INI-files depend heavily on the used INI-parser and its configuration.
+
+This is the attempt to create an overview of commonly used parsers and their interpretation of different INI-dialects to find a common feature-set which may be used as base for the strict specification of the EditorConfig-INI parser.
```

Comment:
Are you suggesting that we need to revisit the EditorConfig INI file format specification to support these baseline features?